### PR TITLE
[all]Create learnAddCommits

### PR DESCRIPTION
**Pull request by CharlieRyder**
sfe

### CODE REVIEW

File: learnAddCommits [hunk 1]
```diff
@@ -0,0 +1 @@
+tweset
```
Comment (charlie-yi) [inline learnAddCommits:1]: whats

Comment (charlie-yi) [inline learnAddCommits:1]: serfwear

**Review by charlie-yi — COMMENTED** [review]
safer

File: learnAddCommits (hunk 1 above)
Comment (CharlieRyder) [reply learnAddCommits:1]: sdf

Comment (CharlieRyder) [reply learnAddCommits:1]: sfew

File: pullRequest.txt [hunk 2]
```diff
@@ -1 +1,13 @@
...
+safds
+
+asf
+ds
+df
+s
+f
+s
+df
+sd
```
Comment (charlie-yi) [inline pullRequest.txt:12]: good job

File: pullRequest.txt [hunk 3]
```diff
@@ -1 +1,13 @@
 test PUll request
+dsfs
```
Comment (charlie-yi) [inline pullRequest.txt:2]: fsdfe

**Review by charlie-yi — APPROVED** [review]

**Pull request merged**